src/components/Select/index.tsx

```diff
@@ -28,11 +39,13 @@ const StyledGrommetSelect = styled(
 `;
 
 // Set button to occupy parent's width, and a default width.
-const Container = styled.div`
+const Container = styled.div(
+	(props: { theme: any; invalid?: boolean }) => `
```

Comment:
```suggestion
	(props: { invalid?: boolean }) => `
```
I am not certain, but I think you don't need to specify theme as a prop (if you do, set the type as `Theme`)